apps/frontend/core-dapp/src/components/Intercom/IntercomButton.tsx

```diff
@@ -0,0 +1,32 @@
+import { FC, ReactNode, Suspense, useEffect } from 'react'
+import { useIntercom } from './index'
+
+type Props = {
+  children: (params: { onClick: () => void }) => ReactNode | undefined
```

Comment:
Probably should enforce return of ReactNode?

```suggestion
  children: (params: { onClick: () => void }) => ReactNode
```